apps/console/src/features/users/components/add-user.tsx

```diff
@@ -324,22 +247,33 @@ export const AddUser: React.FunctionComponent<AddUserProps> = (props: AddUserPro
                                 showPassword={ t("common:showPassword") }
                                 type="password"
                                 value={ isPasswordGenerated ? randomPassword : initialValues?.newPassword }
-                                listen={ handlePasswordChange }
-                                loading={ isPasswordRegExLoading }
-                                validation={ (value: string, validation: Validation) => {
-                                    if (!isPasswordPatternValid) {
+                                validation={ async (value: string, validation: Validation) => {
+                                    let passwordRegex = "";
+                                    if (userStore !== UserstoreConstants.PRIMARY_USER_STORE) {
+                                        // Set the username regEx of the secondary user store.
+                                        passwordRegex
+                                            = await getUserstoreRegEx(
+                                                userStore, USERSTORE_REGEX_PROPERTIES.PasswordRegEx)
+                                    } else {
+                                        // Set the username regEx of the primary user store.
+                                        passwordRegex = PRIMARY_USERSTORE_PROPERTY_VALUES.PasswordJavaScriptRegEx;
+                                    }
+
+                                    if(!validateInputAgainstRegEx(value, passwordRegex)){
                                         validation.isValid = false;
                                         validation.errorMessages.push( t("adminPortal:components.user.forms." +
-                                            "addUserFor1m.inputs.newPassword.validations.regExViolation") );
+                                            "addUserForm.inputs.newPassword.validations.regExViolation") );
                                     }
 
-                                    if (passwordScore < 3) {
+                                    // Password strength validation is disabled
```

Comment:
Why is it removed? No longer required? If so shall we remove the code out?

Reply:
Resolved by ff6c234